pkgs/characters/pubspec.yaml

```
@@ -12,5 +12,5 @@ environment:
   sdk: ^3.4.0
 
 dev_dependencies:
-  dart_flutter_team_lints: ^3.0.0
+  dart_flutter_team_lints: ^3.2.0
```

From the CI error - `Because characters depends on dart_flutter_team_lints >=3.2.0 which requires SDK version >=3.5.0 <4.0.0, version solving failed` - I'd leave this dep at ^3.0.0:

```suggestion
  dart_flutter_team_lints: ^3.0.0
```

---

Bigger isn't better, who'd have thunk? (At least we know we're prepared.)